src/docs.js

```diff
@@ -163,14 +176,20 @@ async function downloader(options) {
     credentials: options.credentials,
   });
 
-  let creds = await auth.awsS3Credentials('read-only', options.bucket, options.project + '/');
+  let creds = await auth.awsS3Credentials(
+    'read-only',
+    options.bucket,
+    options.project + '/'
+  );
```

Comment:
It looks like this -- and a few other places in this file -- are just reformatting code without changing it?  It's usually best to put such changes in their own commit, and preferably in their own PR.  Otherwise, the reviewer (me!) spends a bunch of time trying to figure out what changed and why, when really nothing important changed.

If your editor is doing this sort of reformatting, it's best to disable that.  We have an Eslint configuration for Taskcluster source code that handles the conventions we are most concerned with.  Everything else is at the preference of the original author, and shouldn't be changed.